wled00/wled_server.cpp

```diff
@@ -448,6 +448,49 @@ void initServer()
     request->send(200, FPSTR(CONTENT_TYPE_PLAIN), (String)millis());
   });
 
+  // Endpoint to get version-info.json
+  server.on(F("/version-info.json"), HTTP_GET, [](AsyncWebServerRequest *request){
+    File file = WLED_FS.open("/version-info.json", "r");
+    if (!file) {
+      // Create default version-info.json if it doesn't exist
+      request->send(200, FPSTR(CONTENT_TYPE_JSON), F("{\"version\":\"\",\"neverAsk\":false}"));
+      return;
+    }
+    request->send(file, "/version-info.json", FPSTR(CONTENT_TYPE_JSON));
+    file.close();
+  });
```

Comment:
Do we have new dedicated endpoint, hijack the file editor's (/edit?func=edit&path=version-info.json) , or add to the existing /info endpoint @willmmiles ?